packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -200,11 +200,8 @@ export function ArticleEditor({id}: ArticleEditorProps) {
 
   function countTitle(blocks: BlockValue[]) {
     return blocks.reduce((charLength: number, block: BlockValue) => {
-      if (block.type === BlockType.Title)
```

Comment:
i don't think this file should be changed.

Reply:
same as above